services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/dto/TenantConfig.java

```diff
@@ -0,0 +1,63 @@
+package com.sourcefuse.jarc.services.usertenantservice.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sourcefuse.jarc.services.usertenantservice.commons.UserModifiableEntity;
+import com.sourcefuse.jarc.services.usertenantservice.enums.ConfigKey;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import java.io.Serializable;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "tenant_configs", schema = "main")
+public class TenantConfig extends UserModifiableEntity implements Serializable {
```

Comment:
why Serializable

Reply:
Serialization reason mentioned above ,but if requirement is different we can remove it.